WordPress/Classes/ViewRelated/Blog/SharingAccountViewController.swift

```diff
@@ -72,6 +81,25 @@ import WordPressShared
     }
 
 
+    fileprivate func showNoResultsViewController() {
+        let title = NSLocalizedString("No Accounts Found",
```

Comment:
This is a completely personal preference, but I tend to declare strings as constants (or as static cases in an enumeration), so that, at the call site, like here, it would read similar to this:

````
fileprivate func showNoResultsViewController() {
         noResultsViewController.configure(title: "", buttonTitle: NoResults.buttonTitle, subtitle: NoResults.message, image: nil, accessoryView: nil)
         noResultsViewController.delegate = self
}
````

I have found that makes the code slightly easier to parse for me. What do you think?

Reply:
I'm a fan of that when a string is reused.  When used only once I like declaring a string in the context of its usage as a way of having as much context as possible in one place.